src/Uniphar.Dotnet.Cli/Uniphar.Dotnet.Cli.csproj

```diff
@@ -0,0 +1,28 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+	<PropertyGroup>
+		<OutputType>Exe</OutputType>
+		<TargetFramework>net9.0</TargetFramework>
+		<RootNamespace>Uniphar.Dotnet.Cli</RootNamespace>
+		<ImplicitUsings>enable</ImplicitUsings>
+		<Nullable>enable</Nullable>
+		<PackAsTool>true</PackAsTool>
+		<ToolCommandName>Uniphar</ToolCommandName>
+		<PackageOutputPath>./nupkg</PackageOutputPath>
+		<PackageVersion>1.0.0</PackageVersion>
+		<Authors>Uniphar</Authors>
+		<Title>Uniphar.Dotnet.Cli</Title>
+		<Description>The repository for Uniphar.Dotnet.Cli, a .NET CLI tool designed to obtain tokens from an Entra application. Easily accessible via NuGet, this tool is built to support and simplify your development and testing processes. More features will be added in future updates.</Description>
```

Comment:
Maybe make the description a little more generic so we don't have to update it in the future if/when we add new tools? e.g. "An internal tool to streamline some everyday development and testing practices at Uniphar."